src/storm/solver/IterativeMinMaxLinearEquationSolver.cpp

```diff
@@ -221,8 +221,6 @@ bool IterativeMinMaxLinearEquationSolver<ValueType, SolutionType>::solveInducedE
         STORM_LOG_ASSERT(subB.size() == x.size(), "Sizes of subB and x do not coincide.");
         STORM_LOG_ASSERT(this->linearEquationSolverFactory != nullptr, "Wrong constructor was called.");
 
-        bool convertToEquationSystem = this->linearEquationSolverFactory->getEquationProblemFormat(env) == LinearEquationSolverProblemFormat::EquationSystem;
```

Comment:
This seems relevant as it controls whether we have to build an equation system of the form `Ax=b` (e.g. gmm, eigen) or of the form `Ax+b=x` (e.g. topological solver, native).

As far as I can see, the code below always builds `Ax+b=x`. I suggest to change this to

```
STORM_LOG_THROW(this->linearEquationSolverFactory->getEquationProblemFormat(env) == LinearEquationSolverProblemFormat::FixedPointSystem, storm::exceptions::NotImplementedException, "Solving induced system of Interval Model not supported for the selected equation solver");
```

And create an issue for that?